README.md

```diff
@@ -402,6 +402,27 @@ aa('viewedFilters', {
 | `eventName` | `string`   | The name of the event                                            |
 | `filters`   | `string[]` | The list of filters that was clicked as `'${attr}${op}${value}'` |
 
+### Batch Events
+
+You can send multiple events in a single HTTP request, by using `sendEvents` method.
+
+```js
+aa('sendEvents', [
+  {
+    eventType,
+    eventName,
+    userToken,
+    ...
+  }
+]);
+```
+
+| Option      | Type                            | Description                                    |
+| ----------- | ------------------------------- | ---------------------------------------------- |
+| `eventType` | `view`                          | `click` | `conversion` | The name of the index related to the event     |
+| `eventName` | `string`                        | The name of the event                          |
+| `userToken` | `string` (optional)             | You can specify userToken per event. |
```

Comment:
this option is required on the doc, is it different on this lib?

Reply:
Anonymous user token or the one set by `aa('setUserToken', ...)` will be used by default, but they can override it by providing explicit user token within the event object.

This is the same as,

```js
aa('init', { ..., useCookie: true });

aa('clickObjectIDs', {
  ....
  userToken: 'something-explicit-here'
});
```

Do you think more explanation should be there?

Reply:
Ok make sense! I think it would be nice to have it here indeed